code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -6,7 +6,9 @@
 	. += ..() + config_human_delay + dna.species.movement_delay(src)
 
 /mob/living/carbon/human/slip(knockdown_amount, obj/O, lube)
-	if(isobj(shoes) && (shoes.flags_1&NOSLIP_1) && !(lube&GALOSHES_DONT_HELP))
+	if(has_trait(TRAIT_NOSLIPALL))

```

Comment:
where do you grant this trait ever

Reply:
NOSLIPALL is not granted, but it is here for future use.